webapp/src/components/App/Singlestudy/explore/Xpansion/Settings/SettingsForm.tsx

```diff
@@ -217,21 +218,8 @@ function SettingsForm(props: PropType) {
             onChange={(e) =>
               handleChange("log_level", parseInt(e.target.value, 10))
             }
-            sx={{ mb: 1 }}
+            sx={{ mb: 1, width: 1 }}
```

Comment:
you can remove all  the `width: 1` and add it on the parent only one time to avoid repetition 

```javascript
 "&> div": {
              mr: 2,
              mb: 2,
              flexGrow: 1,
              width: 1
            },
```
(see the previous example comment)